README.md

```diff
@@ -238,15 +226,15 @@ $ make lint
 
 When developing, you can use the `Makefile` for doing the following operations:
 
-| Name               | Description                                            |
-| ------------------:| -------------------------------------------------------|
-| `install`          | Install the Kong luarock globally                      |
-| `dev`              | Install development dependencies                       |
-| `lint`             | Lint Lua files in `kong/` and `spec/`                  |
-| `test`             | Run the unit tests suite                               |
-| `test-integration` | Run the integration tests suite                        |
-| `test-plugins`     | Run the plugins test suite                             |
-| `test-all`         | Run all unit + integration + plugins tests at once     |
+|               Name | Description                                        |
+| -----------------: | -------------------------------------------------- |
+|          `install` | Install the Kong luarock globally                  |
+|              `dev` | Install development dependencies                   |
+|             `lint` | Lint Lua files in `kong/` and `spec/`              |
+|             `test` | Run the unit tests suite                           |
+| `test-integration` | Run the integration tests suite                    |
+|     `test-plugins` | Run the plugins test suite                         |
+|         `test-all` | Run all unit + integration + plugins tests at once |
```

Comment:
not sure if it has any effect or not.  if not, it only makes it harder to maintain.